### PR TITLE
CFE-463: Change the version from v1alpha1 to v1alpha2

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.12+

Issue: https://issues.redhat.com/browse/CFE-463

Link to docs preview:
http://file.rdu.redhat.com/sdudhgao/node-observability/scalability_and_performance/node-observability-operator.html#running-profiling-query_node-observability-operator

http://file.rdu.redhat.com/sdudhgao/node-observability/scalability_and_performance/node-observability-operator.html#creating-node-observability-custom-resource_node-observability-operator

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by xenolinux** [comment]
@sunilcio Could you please take a look at this PR? Thanks

File: modules/node-observability-create-custom-resource.adoc [hunk 1]
```diff
@@ -45,13 +45,13 @@ $ oc project node-observability-operator
...
 ----
-    apiVersion: nodeobservability.olm.openshift.io/v1alpha1
+    apiVersion: nodeobservability.olm.openshift.io/v1alpha2
     kind: NodeObservability
     metadata:
       name: cluster <1>
     spec:
       nodeSelector:
-        node-role.kubernetes.io/worker: "" <2>
+        kubernetes.io/hostname: <node_name> <2>
```
Comment (alebedev87) [inline modules/node-observability-create-custom-resource.adoc:54]:
```suggestion
        kubernetes.io/hostname: <node_hostname> <2>
```

**Comment by Amrita42** [comment]
branch/enterprise-4.12

**Comment by Amrita42** [comment]
@xenolinux can you please check **Installing the Node Observability Operator using the CLI** step 4 mentions API version as v1alpha1 . I think it should be  v1alpha2.

**Comment by xenolinux** [comment]
> @xenolinux can you please check **Installing the Node Observability Operator using the CLI** step 4 mentions API version as v1alpha1 . I think it should be v1alpha2.

@Amrita42 SME had confirmed that it should be as is since the subscription uses the v1alpha1 version.

**Comment by Amrita42** [comment]
Thanks for that clarification @xenolinux

**Comment by Amrita42** [comment]
lgtm

**Review by bergerhoffer — COMMENTED** [review]
Looks good for merging

**Pull request merged**